.github/workflows/on-chain-upgrade.yml

```diff
@@ -22,9 +22,9 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
-      GENESIS_DESMOS_VERSION: "v2.3.0-mainnet"
-      GENESIS_URL: "https://raw.githubusercontent.com/RiccardoM/desmos-states/master/desmos-mainnet-1160425.json"
-      UPGRADE_NAME: "v2.3.1"
+      GENESIS_DESMOS_VERSION: "v2.3.1"
```

Comment:
This has been edited in order to test an on-chain-upgrade with the migration of the `ApplicationLinks` store and `params`